### ISSUE TITLE
Unable to install any package

### ISSUE THREAD

**Issue by yousifamer** [LykosAI/StabilityMatrix#586]
i am new to stability matrix, i have not been able to install any package because of the following issue:

Unpacking resources
Unpacking resources
Could not install stable-diffusion-webui (StabilityMatrix.Core.Exceptions.ProcessException: Process git exited with code 128. {StdOut = , StdErr = Cloning into 'D:\SD\Data\Packages\Stable Diffusion WebUI'...
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
error: 1886 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
}
   at StabilityMatrix.Core.Processes.ProcessResult.EnsureSuccessExitCode()
   at StabilityMatrix.Avalonia.Helpers.WindowsPrerequisiteHelper.RunGit(ProcessArgs args, String workingDirectory)
   at StabilityMatrix.Core.Models.Packages.BaseGitPackage.DownloadPackage(String installLocation, DownloadPackageVersionOptions versionOptions, IProgress`1 progress)
   at StabilityMatrix.Core.Models.PackageModification.PackageModificationRunner.ExecuteSteps(IReadOnlyList`1 steps))

The issue happens with all packages

Operating System: Windows 11

**Comment by Giribot** [comment]
Hello, I'm not a programmer but there are two error messages you sent that might help:
The error “error: 1886 bytes of body are still expected” and “fatal: early EOF” are issues that occur when a Git operation expects data that has not been fully received or processed. This could be due to network interruptions or communication problems between Git and the remote server (poor communication via Wifi? micro-cuts in your internet connection? have you tried to download all of this via a network cable?), or a storage memory full (not enough space on the media to download all the files).

Good day to you

**Comment by gab-luz** [comment]
Here, on my computer, I've removed the package's folder and tried again and it works!

**Comment by alanng90211** [comment]
> Here, on my computer, I've removed the package's folder and tried again and it works!

hi there, where can i find the packages folder?

**Comment by ansaus** [comment]
> > Here, on my computer, I've removed the package's folder and tried again and it works!
> 
> hi there, where can i find the packages folder?

it works. you should look for Data\Packages folder inside installation folder or Users\*CurUser*\AppData\Local...

**Comment by alanng90211** [comment]
Thanks!

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by BrechtCorbeel** [comment]
I got it to work after turning off output sharing in StabilityMatrix as a prerequisite before install.

**Comment by ionite34** [comment]
Marking as not planned for now, not sure how to reproduce or what we can do to resolve the git connectivity issues atm.

**Comment by mk-ultron** [comment]
same issue here. Can't update any package, stability matrix just crashes when I try to launch any package. Tried disabling output sharing and even removed the package folder, but the same issue happened.

**Comment by BrechtCorbeel** [comment]
> same issue here. Can't update any package, stability matrix just crashes when I try to launch any package. Tried disabling output sharing and even removed the package folder, but the same issue happened.

I still have the issue so no update on any fix from me.

**Comment by chibikru** [comment]
Try download from github "stable-diffusion-webui-master.zip" and unpack with folder "stable-diffusion-webui-master" to Packages . Usually its "...\AppData\Roaming\StabilityMatrix\Packages\..."

**Comment by soul0710** [comment]
I fixed it by downloading a model before installing the package

**Comment by phillism** [comment]
I fixed it by downloading the ZIP package manually at GitHub and moving to the package folder.

**Comment by dajohns5** [comment]
I have an issue that's semi-close to this. I'm attempting to install clip_vision_h The other day, I tried a bunch of different ones, some had apache, some had MIT listed in them. I'm running Windows 11. 32gb of memory, plenty of disk space, hardware never has an issue, but the generation can sometimes be slow.

My issue is that when I try to download the clips, they go into the queue and then nothing. No progress, they stay at 0% until I eventually close the program. I tried download one clip package manually, but no matter where I put it, it doesn't recognize that it's there. I'd appreciate any suggestions or tips.
